internal/cli/main.go

```diff
@@ -20,6 +20,7 @@ const defaultTimeFormat = "20060102150405"
 // set main log
 var log = &Log{}
 
+// Main main function of a cli application. It is publiic for backwards compatibility with `cli` package
```

Comment:
Make this `Main function` or `Main is the main function`

Reply:
And `public` is misspelled

Reply:
fixed in https://github.com/r3code/migrate/commit/f20abab955d9f8c80ad72c49b473bda834a486f5 and  https://github.com/r3code/migrate/commit/47c2c2c834d98d9f0eac914ab60098a81ae40819